docker-compose.yml

```diff
@@ -15,6 +15,7 @@ services:
       - TRAVIS_PULL_REQUEST=${TRAVIS_PULL_REQUEST}
       - REVIEWDOG_GITHUB_API_TOKEN=${REVIEWDOG_GITHUB_API_TOKEN}
       - PERCONA_TEST_DBAAS=${PERCONA_TEST_DBAAS:-0}
+      - PERCONA_TEST_IA=1
```

Comment:
should we set 1 here? let's give ability to override it with ENV Var

Reply:
Unlike DBaaS, IA does not require anything else, so there is no use for that configurability. That environment variable will be removed after 2.12 release. We have it _just_ to allow us to merge some PRs now